### PR TITLE
Speak of "personal" intercepts, not "selective" intercepts

### PR DESCRIPTION
**Pull request by LukeShu**
There are 2 commits of interest here, and then a bunch of noise with syncing between versions and repos:
- https://github.com/telepresenceio/telepresence.io/commit/888b77ca678508a1c8201c1a661585978788fdfb
- https://github.com/telepresenceio/telepresence.io/commit/2945c5e58aee39e020e63a4ccf1bb78a227f02b3

This has a pair of counterparts: https://github.com/datawire/ambassador-docs/pull/419 https://github.com/datawire/ambassador-docs/pull/420

### CODE REVIEW

File: docs/v2.4/reference/config.md [hunk 1]
```diff
@@ -65,21 +79,21 @@ to handle installation of the `traffic-agents`.
...
-|`agentImage`|$registry/$imageName:$imageTag to use when installing the Traffic Agent. Changing this value will update pre-existing `traffic-agents` to use this new image. * the `registry` value is not used for the `traffic-agent` if you have this value set *||
-|`webhookRegistry`|The container $registry that the [Traffic Manager](../cluster-config/#mutating-webhook) will use with the `webhookAgentImage` *This value is only used if a new traffic-manager is deployed*||
-|`webhookAgentImage`|The container image that the [Traffic Manager](../cluster-config/#mutating-webhook) will use when installing the Traffic Agent in annotated pods *This value is only used if a new traffic-manager is deployed*||
+| Field               | Description                                                                                                                                                                                                                                                                                                                                                                                    | Default              |
```
Comment (donnyyung) [inline docs/v2.4/reference/config.md:82]: I know they are all strings, but just for the sake of uniformity, should we include a type column here as well?

File: docs/v2.4/reference/config.md [hunk 2]
```diff
@@ -92,19 +92,26 @@ These fields control how the client interacts with the Cloud service.
...
-| `skipLogin`       | Whether the cli should skip automatic login to Ambassador Cloud. If set to true, you must have a [license](../cluster-config/#air-gapped-cluster) installed in the cluster in order to be able to perform selective intercepts             | [bool][yaml-bool]                          | false   |
+| `skipLogin`       | Whether the CLI should skip automatic login to Ambassador Cloud.  If set to true, in order to perform personal intercepts you must have a [license key](../cluster-config/#air-gapped-cluster) installed in the cluster.                   | [bool][yaml-bool]                          | false   |
 | `refreshMessages` | How frequently the CLI should communicate with Ambassador Cloud to get new command messages, which also resets whether the message has been raised or not. You will see each message at most once within the duration given by this config | [duration][go-duration] [string][yaml-str] | 168h    |
 
-Telepresence attempts to auto-detect if the cluster is air-gapped,
-be sure to set the `skipLogin` value to `true`
+Telepresence attempts to auto-detect if the cluster is capable of
+communication with Ambassador Cloud, but may still prompt you to log
```
Comment (donnyyung) [inline docs/v2.4/reference/config.md:99]: Super nit so can ignore: Perhaps change `log in in` to `login in` so we don't have the `in` repetition.

File: docs/v2.4/reference/intercepts.md [hunk 3]
```diff
@@ -2,28 +2,103 @@ import Alert from '@material-ui/lab/Alert';
...
+`--preview-url=true`.  If you hadn't been logged in it would have
+defaulted to `--preview-url=false`.  This tells Telepresence to take
+advantage of Ambassador Cloud to create a preview URL for this
+intercept, creating a shareable URL that automatically sets the
+appropriate headers to have requests coming from the preview URL be
+intercepted.  In order to create the preview URL, it will prompt you
+for four settings about how your cluster's ingress is configured.  For
+each, Telepresence tries to intelligently detect the correct value for
+your cluster; if it detects it correctly, may simply press "enter" and
+accept the default, otherwise you must tell Telepresence the correct
```
Comment (donnyyung) [inline docs/v2.4/reference/intercepts.md:68]: the correct value.*

File: docs/v2.4/reference/intercepts.md [hunk 4]
```diff
@@ -112,9 +177,17 @@ Finally, run `telepresence leave <name of intercept>` to stop the intercept.
...
+If you are trying to intercept a service that has multiple ports, you
+need to tell Telepresence which service port you are trying to
+intercept.  To specify, you can either use the name of the service
+port or the port number itself.  To see which options might be
+available to you and your service, use kubectl to describe your
+service or look in the object's YAML. For more information on multiple
+ports, see the [Kubernetes documentation][kube-multi-port-services].
 
-```
+[kube-multi-port-services]: https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services
```
Comment (donnyyung) [inline docs/v2.4/reference/intercepts.md:188]: Is the purpose of this to prevent long lines (e.g. on like 186) or is there some other significance?

**Review by donnyyung — CHANGES_REQUESTED** [review]
A nit or two and a few things we should change / fix. But otherwise LGTM

File: docs/v2.4/reference/intercepts.md (hunk 4 above)
Comment (LukeShu) [reply docs/v2.4/reference/intercepts.md:188]: Yeah, just to make it more readable in a text editor.

File: docs/v2.4/reference/config.md (hunk 2 above)
Comment (LukeShu) [reply docs/v2.4/reference/config.md:99]: "login" is the noun form, "log in" is the verb form.

File: docs/v2.4/reference/intercepts.md (hunk 3 above)
Comment (LukeShu) [reply docs/v2.4/reference/intercepts.md:68]: Fixed.

File: docs/v2.4/reference/config.md (hunk 1 above)
Comment (LukeShu) [reply docs/v2.4/reference/config.md]: Done.

**Review by donnyyung — APPROVED** [review]

**Pull request merged**